src/main/java/org/rumbledb/cli/JsoniqQueryExecutor.java

```diff
@@ -273,10 +275,12 @@ private String getIteratorOutput(RuntimeIterator iterator) {
                         this.configuration.getResultSizeCap() == 0)
             ) {
                 sb.append(iterator.next().serialize());
-                sb.append("\n");
+                // sb.append("\n");
+                sb.append(", ");
                 itemCount++;
             }
-            // remove last comma
+            sb.setLength(sb.length() - 2); // remove last comma and space
+            sb.append(")");
             return sb.toString();
```

Comment:
Changes in this file will be reverted once all tests are prepared. This serialization makes test output preparation easier.